src/action.ts

```diff
@@ -236,7 +238,7 @@ class Action {
     const artifacts: string[] = []
 
     const outputsDir = path.join(this.root, "outputs", randomUUID())
```

Comment:
This is the directory you want to create within the `tmp` dir:

```suggestion
    const outputsDir = fs.mkdtempSync("vib-action")
```

You shouldn't need to modify the existing `mkdir` function.